### DISCUSSION TITLE
Dynamic power options like in Crossfire

### DISCUSSION THREAD

**Discussion by kristjanbjarni** [ExpressLRS/ExpressLRS#1949] [Ideas]
I find the current dynamic power using AUX switch in ELRS to be very unwieldy. The way it's done in Crossfire is much better in my opinion. The problem is that I want to use low power when racing and indoors but maybe use higher power when doing freestyle or long range. Currently in ELRS I have to remember each time I change styles is to go into the ELRS adapter menu and change max power accordingly. The way this is done in Crossfire is with a 3 position switch with the maximum power for each position. That way the default switch position with Crossfire is set to low but I can just flip a switch to change it to two other maximum power outputs without having to go into the menu every time. This way you also change the max power option at any time in flight which is not possible with ELRS.

**Comment by pkendall64** [comment]
Why don't you setup different models (with a different receiver number) in Edge/OpenTX but not enable model match.
That allows you to have different models for indoor, racing, freestlye, LR etc, but use the same physical model.

See https://www.expresslrs.org/3.0/software/model-config-match/

**Reply by kristjanbjarni** [reply]
Well that is possible, but personally I feel that is just to much work for just this simple thing. I like to keep a single model config for each TX protocol. My model configuration is very complex with lot of audio notifications and telemetry screens. It would mean I would have to maintain all changes for each model configuration.
